open_tms_backend/open_tms_update_ticket/index.js

```diff
@@ -9,10 +9,15 @@ const handler = async (event) => {
     return sendResponse(res, 200, { message: 'Health check success' })
   }
 
-  /**
-   * Add update ticket logic here
-   */
+  // get the ticket ID to update
+  const ticketId = req.params.ticketId
 
```

Comment:
Update ticket has to be a separate function block which is not part of a shared block